functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,116 @@
+function Remove-DbaDbView {
+    <#
+    .SYNOPSIS
+        Removes a database view(s) from each database and SQL Server instance.
+
+    .DESCRIPTION
+        Removes a database view(s), with supported piping from Get-DbaDbView.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
```

Comment:
This text has to be changed, as the parameter does not accept pipeline input.

Reply:
I'll fix this as well.

Reply:
We may have to fix this in other remove commands too in another PR